testdata/imports_options.proto

```diff
@@ -15,6 +17,7 @@ message Channel2 {
   string name = 2 [(personal) = true, (owner) = 'test'];
   string description = 3;
   map<string, int32> map = 4 [(personal) = true];
+  string address = 5 [(custom_options) = { personal: true, internal: false, owner: "some owner" }];
```

Comment:
Out of curiosity, are the `,` separators in the map required? They don't seem to be used in your other examples (line 8 above in this file), but they are used here.

Adding a test case for the options both with and without commas would be great if both ways are valid.

Reply:
Will add the test